tests/sweep_framework/sweeps/data_movement/squeeze/squeeze_pytorch2.py

```diff
@@ -0,0 +1,119 @@
+# SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Optional, Tuple
+
+import torch
+import random
+import ttnn
+
+from tests.ttnn.utils_for_testing import check_with_pcc, start_measuring_time, stop_measuring_time
+from models.utility_functions import torch_random
+
+TIMEOUT = 10
+# seed for random
+random.seed(0)
+
+
+# Create several shape types, such as [A, B, C, D] where any may or may not be included and any value ranges from 1-x
+# for i -> rand(1,4):
+#   shape should be [rand between 1,4096 for each N]
+#   One index (0->n) should have length/value 1 to squeeze, index i #this should also be random
+# dim = i or -(length-i) 50% chance here
+# All can go in a for loop for how many samples you want
+# This will do every combination of the following dtype and layout, so we will address RM and bfloat8 in invalidate vector
+def generate_squeeze_config(num_samples=10):
```

Comment:
I see this is commented out in parameters... are all of these the functions that we're used to generate the params? Leaving the code for future reference?

Reply:
I'm leaving all these generation functions in for when we do adversarial testing!